### PR TITLE
Update Catch to 1.10.0

### PR DESCRIPTION
**Pull request by horenmar**

### CODE REVIEW

File: ports/catch/portfile.cmake [hunk 1]
```diff
@@ -1,16 +1,15 @@
 include(vcpkg_common_functions)
 
-set(CATCH_VERSION v1.9.7)
-vcpkg_download_distfile(HEADER
+set(CATCH_VERSION v1.10.0)vcpkg_download_distfile(HEADER
```
Comment (KindDragon) [inline ports/catch/portfile.cmake]: New line?

Comment (horenmar) [reply ports/catch/portfile.cmake]: Good catch, wonder what made the script break.

**Comment by horenmar** [comment]
Fixed the missing newline and fixed our update script, so it isn't slowly losing newlines anymore.

**Comment by alexkaratarakis** [comment]
Thanks!

**Pull request merged**